airflow/utils/log/file_task_handler.py

```diff
@@ -351,6 +351,8 @@ def _read(
         ti: TaskInstance,
         try_number: int,
         metadata: dict[str, Any] | None = None,
+        offset: int = 0,
+        limit: int = 100,
```

Comment:
The defaults for these should be None, and by default there should be no pagination. There should only be pagination when it's specifically configured.

Reply:
Ah, I see, so if they didn't configure it, there would be pagination? Would a simple fix make them an optional parameter?If Null it'll ignore all the pagination logic. Else it'll use these default values?

Reply:
> Ah, I see, so if they didn't configure it, there would be pagination?

If they didn't configure it, there would be _no_ pagination. It needs to be opt in.